README.md

```diff
@@ -567,6 +569,9 @@ To disable liveness or readiness probe, set value of `enabled:` to `false`.
 
 All notable changes to this project will be documented here
 
+### v1.3.0
```

Comment:
@aslafy-z Why should we bump minor? Wouldn't bumping the patch version will be enough? v1.2.8

Reply:
Done